apps/member/src/components/group/ActivityAssignmentEditor/ActivityAssignmentEditor.tsx

```diff
@@ -58,13 +61,18 @@ const ActivityAssignmentEditor = ({ parentId, activityGroupId }: Props) => {
       },
       files: file ? formData : undefined,
     });
+    await setBoard(defaultBoard);
```

Comment:
`setBoard`까지 `await` 해줘야할 이유가 있을까요?
`activityGroupBoardMutate` 완료 이후 실행되는 로직이니 해당 `await` 문은 불필요해보입니다.

```suggestion
    setBoard(defaultBoard);
```

Reply:
추가로 `mutate`에서 `onSuccess`를 사용하여 사이트 이펙트를 핸들링하면 좋을 거 같아요.

Reply:
[974f843](https://github.com/kgu-clab/clab-platforms/pull/221/commits/974f8434fb4096f0d43176c1d3adec564d536dff) 해당 부분 동의하여 수정하였습니다!